packages/offix-offline/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "offix-offline",
```

Comment:
@StephenCoady @darahayes Do you like to change package.json name? 
Let me know beforehand :D

Reply:
Its fine with me anyway.